lib/jobs/ansible-job.js

```diff
@@ -65,12 +86,29 @@ function ansibleJobFactory(
             return self._publishAnsibleResult(self.routingKey, data);
         })
         .then(function() {
+            return fs.readFileAsync(self.outFileName, 'utf8');
+        })
+        .then(function(contents) {
+            assert.arrayOfString(self.context.ansibleResultFile);
+            self.context.ansibleResultFile.push(contents);
+            fs.unlink(self.outFileName, function(err) {
```

Comment:
It doesn't matter if the unlink does not complete before the workflows exits, so I did not use the promise there.  Should we still use the promise anyway?